css/filmstrip/_tile_view.scss

```diff
@@ -45,7 +46,18 @@
         position: fixed;
         top: 0;
         width: 100%;
-        z-index: $filmstripVideosZ
+        z-index: $filmstripVideosZ;
+        margin-left: 0px;
+        transition: margin 0.5s, width 0.5s;
+
+        &.shift-right {
+            margin-left: $sidebarWidth;
+            width: calc(100% - #{$sidebarWidth});
+
+            #filmstripRemoteVideos {
+                width: calc(100vw - #{$sidebarWidth});
```

Comment:
why `100vw` and not `100%`?

Reply:
100vw is the width initially set on `#filmstripRemoteVideos` and I thought I'd preserve that, but I can change it if you consider it's more appropriate.

Reply:
No that's fine! Make sense!